project proposal.ipynb

```diff
@@ -77,10 +77,10 @@
     "We almost lost hope - but eventually we managed to find two interesting sources of data. \n",
```

Comment:
<p>if you wanna mention next steps on this specific work (apples direction), then it would be:</p><ol><li>check, on multiple RFs (and not just the best), if the further the prediction from 0, the better the accuracy.</li><li class="ql-indent-1">if so, try modeling RFs with different loss function - one which punish more heavily on predictions that are close to 0.</li><li>extract features importance from RFs.</li><li>create more (different) features.</li><li>repeat the same process on other stocks, and trying to collaborate the different models into 1 general model for all stocks - maybe the general model will predict better for each individual stock.</li></ol>
<br/>

 _Reply via <a href='https://app.reviewnb.com/alongouldman/stock_market_neural_nets/pull/9/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='81'/>